docs/en/serving/api_server.md

```diff
@@ -173,6 +173,10 @@ for message in messages:
         print(item)
 ```
 
+### Tools
```

Comment:
do we need to add the md file to `index.rst` since pr #1880 is merged. May need to sync with main branch.